Silicon/CommonSocPkg/Library/GpioV2Lib/GpioV2Init.c

```diff
@@ -0,0 +1,1485 @@
+/** @file
+  This file contains routines for GPIO
+
+  Copyright (c) 2020 - 2023, Intel Corporation. All rights reserved.<BR>
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#include <Uefi/UefiBaseType.h>
+#include <GpioConfig.h>
+#include <Library/GpioV2Lib.h>
+#include <Library/GpioSiLib.h>
+#include <Library/PchSbiAccessLib.h>
+#include <Library/ConfigDataLib.h>
+#include <Include/GpioV2Config.h>
+#include <GpioV2PinsMtlSoc.h>
+#include <Library/PrintLib.h>
+#include <Library/MtlSocGpioTopologyLib.h>
```

Comment:
MtlSocGpioTopologyLib.h and GpioV2PinsMtlSoc.h should not be used in the common library.